celery/utils/collections.py

```diff
@@ -573,9 +573,15 @@ def clear(self):
     def add(self, item, now=None):
         # type: (Any, float) -> None
         """Add a new item, or reset the expiry time of an existing item."""
-        now = now or time.time()
+        now = now or monotonic()
```

Comment:
`now` should be equal to the diff between the time the LimitedSet was initialized and the current value.

Reply:
We need to address this before merging

Reply:
@thedrow isn't generating a unique value here the requirement? `monotonic` achieves just that.